lizzy/apps/senza.py

```diff
@@ -1,17 +1,20 @@
 from typing import Optional, List, Dict
 import tempfile
 
+from ..version import VERSION
```

Comment:
I think we should change the version to match the one in setup.py (or even import lizzy.version from setup.py though that breaks coverage reporting for that file).
If you decided to do so please change it also on the lizzy-client.